### ISSUE TITLE
about model.py yolo_eval() function

### ISSUE THREAD

**Issue by Amerry-c** [qqwweee/keras-yolo3#103]
for c in range(num_classes):
        # TODO: use keras backend instead of tf.
        class_boxes = tf.boolean_mask(boxes, mask[:, c])
        class_box_scores = tf.boolean_mask(box_scores[:, c], mask[:, c])
        nms_index = tf.image.non_max_suppression(
            class_boxes, class_box_scores, max_boxes_tensor, iou_threshold=iou_threshold)
        class_boxes = K.gather(class_boxes, nms_index)
        class_box_scores = K.gather(class_box_scores, nms_index)
        classes = K.ones_like(class_box_scores, 'int32') * c
        boxes_.append(class_boxes)
        scores_.append(class_box_scores)
        classes_.append(classes)
    boxes_ = K.concatenate(boxes_, axis=0)
    scores_ = K.concatenate(scores_, axis=0)
    classes_ = K.concatenate(classes_, axis=0)

    return boxes_, scores_, classes_
  @qqwweee @tanakataiki hi,for some of the above code,when will the version that use keras backend instead of tf come out?

**Comment by qqwweee** [comment]
The repo is tensorflow backend. You have to use tensorflow rather than theano or something else. The answer of your question may be never.

**Comment by Amerry-c** [comment]
@qqwweee ok,thank you for your reply.
